bundler.d/fog.rb

```diff
@@ -1,5 +1,5 @@
 group :fog do
- gem 'fog', '1.25.0'
- gem 'fog-core', '1.25.0'
+ gem 'fog', '1.26.0'
```

Comment:
Minimum is fog 1.26. It has the needed block device mapping v2 to perform the boot from volume creation.